views/make/index.js

```diff
@@ -35,6 +35,10 @@ module.exports = view.extend({
         goBack: function (e) {
             e.preventDefault();
             if (this.$data.mode === 'settings') {
+                app.update({
```

Comment:
So I thought about that, but wasn't really sure the best way to scope a variable to multiple methods. Open to suggestions!